### PR TITLE
Single place for all the tink stack version

### PR DESCRIPTION
**Pull request by gianarb**
Tinkerbell is made of different components as we all know at this point.

Sandbox had those versions all over the places. This PR moves them as
part of the `envrc` file.

As it is today this PR requires an additional flag for the `docker-compose` command as described here #6

### CODE REVIEW

**Review by gauravgahlot — APPROVED** [review]

**Review by thebsdbox — APPROVED** [review]
I'm liking a single place to specific the version and as mentioned [here](https://github.com/tinkerbell/sandbox/issues/6#issuecomment-691983971) I like the move to a `.env` to be more inline with other use-cases.

File: .github/workflows/vagrant-packet.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ jobs:
     - name: Vagrant Test
       run: |
         export VAGRANT_DEFAULT_PROVIDER="virtualbox"
-        go test --timeout 1h -v ./test/_vagrant
+        go test --timeout 1h -v -count 1 ./test/_vagrant
```
Comment (mmlb) [inline .github/workflows/vagrant-packet.yaml:17]: This belongs in its own commit. Actually why is this even needed? The tests should not be cached in GHA right?

**Review by mmlb — CHANGES_REQUESTED** [review]
This is great. I think the variables should be named `*_IMAGE` instead though, since it matches the dc key and you aren't messing with only the version/tag.

File: .github/workflows/vagrant-packet.yaml (hunk 1 above)
Comment (gianarb) [reply .github/workflows/vagrant-packet.yaml:17]: I have added this because apparently they are. I think the self-managed runner we have is caching them

Comment (gianarb) [reply .github/workflows/vagrant-packet.yaml:17]: I will dig into it separately.

Comment (mmlb) [reply .github/workflows/vagrant-packet.yaml:17]: ah!

**Review by mmlb — CHANGES_REQUESTED** [review]
Still should split the `go test` change into its own commit.

**Review by mmlb — APPROVED** [review]
